Dockerfile

```
@@ -17,6 +17,8 @@ ARG VERSION
 ARG VCS_REF
 ARG DOCKERFILE_PATH
 
+USER 10000
```

Let's make sure that the group is also non-zero:
```suggestion
USER 10000:10000
```